armi/reactor/components/componentParameters.py

```diff
@@ -78,13 +78,32 @@ def getComponentParameterDefinitions():
             default=None,
         )
 
+        pb.defParam(
+            "pinNDens",
+            setter=isNumpyArray32Bit("pinNDens"),
+            units=f"#/(bn*{units.CM})",
```

Comment:
For consistency, let's use `atoms` here as well

https://github.com/terrapower/armi/blob/04c06702cec8928fb3b0b5d0e3c3f75f20496622/armi/reactor/assemblyParameters.py#L137

```suggestion
            units=f"atoms/(bn*{units.CM})",
```